shared-module/displayio/EPaperDisplay.h

```diff
@@ -38,7 +38,7 @@ typedef struct {
     displayio_display_core_t core;
     digitalio_digitalinout_obj_t busy;
     uint32_t milliseconds_per_frame;
-    const uint8_t *start_sequence;
```

Comment:
I'm not sure you need to remove this anywhere. I thought you could re-assign another const pointer.